examples/flutter_gallery/lib/welcome/steps/documentation_step.dart

```diff
@@ -0,0 +1,124 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import '../constants.dart';
+import '../welcome_step_state.dart';
+import 'step_container.dart';
+
+const String _kTitle = 'Complete, flexible APIs';
+const String _kSubtitle = 'View full API documentation, when you need it, with a quick tap. Look for the documentation icon in the app bar.';
+
+class DocumentationWelcomeStep extends StatefulWidget {
+  const DocumentationWelcomeStep({Key key}) : super(key: key);
+  @override
+  DocumentationWelcomeStepState createState() => DocumentationWelcomeStepState();
+}
+
+class DocumentationWelcomeStepState extends WelcomeStepState<DocumentationWelcomeStep> with TickerProviderStateMixin {
+
+  Widget _imageWidget;
+  AnimationController _animationController;
+  AnimationController _quickAnimationController;
+  Animation<double> _barScaleAnimation;
+  Animation<double> _barOpacityAnimation;
+  Animation<double> _focusScaleAnimation;
+  Animation<double> _focusOpacityAnimation;
+  Animation<double> _iconOpacityAnimation;
+
+  @override
+  void initState() {
+    super.initState();
+    _configureAnimations();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    _imageWidget ??= imageWidget();
+    return StepContainer(
+      title: _kTitle,
+      subtitle: _kSubtitle,
+      imageContentBuilder: () => _imageWidget
+    );
+  }
+
+  Widget imageWidget() {
```

Comment:
Make function names sound like they cost what they cost. If you create stuff, call it create*